context/app/static/js/components/entity-search/facets/FacetAccordion/style.js

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components';
+import AccordionDetails from '@material-ui/core/ExpansionPanelDetails';
+
+import { StyledAccordionSummary } from 'js/components/searchPage/filters/style';
+
+const InnerAccordionDetails = styled(AccordionDetails)`
+  flex-direction: column;
+  padding: 4px 10px 4px 16px;
+`;
+
+const InnerAccordionSummary = styled(StyledAccordionSummary)`
+  justify-content: left;
+  padding: 0px 16px 0px 16px;
```

Comment:
My sense is that color discrepancies would get to be annoying, and so we put them in theme, but for distances it doesn't make sense to worry about uniformity: The only question is whether one particular component looks right. Is that a good way of framing it?